xrmocap/ops/triangulation/fourdag_triangulator.py

```diff
@@ -0,0 +1,564 @@
+import math
+import numpy as np
+import copy
+import os
+from xrmocap.ops.triangulation import m_shape
+
+###
+n_kps = 19
+shapeSize=10
+joint_parent = [-1, 0, 0, 0, 1, 1, 1, 2, 3, 4, 4, 5, 6, 7, 8, 11, 12, 14, 13]
+hierarchy_map = [0, 0, 0, 0, 1, 1, 1, 1, 1, 2, 2, 2, 2, 2, 2, 3, 3, 3, 3]
+###
+
+def Welsch(c, x):
```

Comment:
It has been defined twice. We should only define it once and import that one when it is needed.